src/main/java/nl/Tech/It/Easy/Tech/It/Easy/TechItEasyApplication.java

```diff
@@ -1,13 +1,15 @@
 package nl.Tech.It.Easy.Tech.It.Easy;
 
+
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
+@ComponentScan(basePackages = {"nl.Tech.It.Easy.Tech.It.Easy.controllers", "nl.Tech.It.Easy.Tech.It.Easy.exceptions"})
```

Comment:
De component scan is een onmisbaar process voor het opstarten van een Spring applicatie, maar dit proces zit al verwerkt in de `@`SpringBootApplication annotatie. Dit is dus een van de verschillen tussen Spring en SpringBoot. Alleen als de packages buiten de standaard mappen structuur (src.main.java.nl.[projectnaam]) vallen heb je dit nodig, maar dan moet je mischien eerst eens afvragen waarom die klassen buiten de standaard mappen structuur staan.